test/WebSites/AjaxAntiForgeryValidation/Startup.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using AjaxAntiForgeryValidation.Models;
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace AjaxAntiForgeryValidation
+{
+    public class Startup
+    {
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                .AddEnvironmentVariables();
+            Configuration = builder.Build();
+        }
+
+        public IConfigurationRoot Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add framework services.
+            services.AddMvc();
+            services.AddAntiforgery(options =>
+            {
+                Console.WriteLine();
+            });
+            services.AddIdentity<ApplicationUser, ApplicationRole>(config =>
```

Comment:
Do we need Identity here? This is pretty heavyweight. Identity is primarily concerned which information about the user, not just whether or not the user is authentic.

Suggest talking to @Tratcher about how we we could just get away with cookie middleware (which Identity uses).